packages/lit-dev-content/samples/examples/properties-custom-converter/index.html

```diff
@@ -11,6 +11,6 @@
 <body>
   <my-element></my-element>
   <hr>
-  <my-element date="Oct 20 2021Z-08:00"></my-element>
+  <my-element date="2021-10-20T00:00:00Z"></my-element>
```

Comment:
Nit: How about just keeping the date without the timezone offset? This seems to work in Firefox.
```suggestion
  <my-element date="Oct 20 2021"></my-element>
```